policyengine_us/parameters/gov/states/ky/tax/income/rate.yaml

```diff
@@ -15,5 +16,10 @@ metadata:
       href: https://revenue.ky.gov/Forms/Form%20740%205-9-23.pdf#page=1
     - title: 2023 Kentucky Estimated Tax Voucher Instructions
       href: https://revenue.ky.gov/Forms/740-ES%20Instructions%203-21-23.pdf
+    - title: Gov Beshear signs income tax cut into law
```

Comment:
please cite this instead, it says signed by governor https://apps.legislature.ky.gov/record/25rs/hb1.html